lib/browser.js

```diff
@@ -1,3 +1,11 @@
+var util = require('./util');
```

Comment:
Absolutely. Webpack actually does polyfill buffer, but I like taking control of which buffer polyfill to use back to us (makes updating when bugs are fixed easier), and to make supporting other tools easier.